Makefile

```diff
@@ -10,6 +10,9 @@ install:
 build:
 	go build cmd/saucectl/saucectl.go
 
+build-linux:
+	GOOS=linux GOARCH=amd64 make build
+
```

Comment:
@alexplischke what do you think on adding something like:

```
build-%:
	GOOS=$* GOARCH=amd64 makd build
```
So we can build for different platforms?